src/stylesheets/main.scss

```diff
@@ -45,12 +46,15 @@
   'components/person',
   'components/article',
   'components/contact',
-  'components/form';
+  'components/form',
+  'components/button',
+  'components/button-empty',
+  'components/button-action';
```

Comment:
It's better to make a scss mixin with this buttons, but I'm going to do it personally tomorrow.